Examples/Io/Root/include/ActsExamples/Io/Root/detail/NuclearInteractionParametrisation.hpp

```diff
@@ -0,0 +1,187 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/EventData/SimParticle.hpp"
+#include "ActsExamples/Utilities/detail/ExtractedSimulationProcess.hpp"
+
+#include <unordered_map>
+
+#include <TH1F.h>
+#include <TVectorF.h>
+
+namespace NuclearInteractionParametrisation {
```

Comment:
If the header is located in the `detail` directory, the contents should also be in the appropriate `detail` namespace, i.e.

```
namespace ActsExamples {
namespace detail {
...
```

Reply:
added